packages/client-core/src/user/components/UserMenu/menus/ProfileMenu.tsx

```diff
@@ -93,13 +93,8 @@ const ProfileMenu = ({ hideLogin, allowAvatarChange, isPopover, changeActiveMenu
     (authState.linkedin && !oauthConnectedState.linkedin) ||
     (authState.twitter && !oauthConnectedState.twitter)
 
-  const removeSocial =
-    (authState?.discord && oauthConnectedState.discord) ||
-    (authState.facebook && oauthConnectedState.facebook) ||
-    (authState.github && oauthConnectedState.github) ||
-    (authState.google && oauthConnectedState.google) ||
-    (authState.linkedin && oauthConnectedState.linkedin) ||
-    (authState.twitter && oauthConnectedState.twitter)
+  /**allow removing social logins if there are at least 2 social logins connected*/
+  const removeSocial = Object.values(oauthConnectedState).filter((value) => value).length > 1
```

Comment:
Forgot to mention this the other day, this should be >= 1. The list of OAuth identity providers that you have connected is the list of identity providers you can remove; there's a check in the client code that will not even attempt to remove the IP if it's the only one. If this were just > 1, then a user with a single connected OAuth IP wouldn't see the IP that they had connected.

Reply:
Right. I have added this change!